sdk/eventhub/azure-eventhub/tests/tracing_common.py

```diff
@@ -0,0 +1,218 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""Fake implementation of AbstractSpan for tests. Copied from `azure-core/tests`."""
+from contextlib import contextmanager
+from azure.core.tracing import HttpSpanMixin, SpanKind
+from typing import Union, Sequence, Optional, Dict
+AttributeValue = Union[
+    str,
+    bool,
+    int,
+    float,
+    Sequence[str],
+    Sequence[bool],
+    Sequence[int],
+    Sequence[float],
+]
+Attributes = Optional[Dict[str, AttributeValue]]
+
+class FakeSpan(HttpSpanMixin, object):
+    # Keep a fake context of the current one
+    CONTEXT = []
+
+    def __init__(self, span=None, name="span", kind=None, *, links=[]):
+        # type: (Optional[Span], Optional[str]) -> None
```

Comment:
we should not have the comment style typing

Reply:
yeah, I plan on fixing all this in the typing PR (which is incoming I promise :P)